apps/backend/src/app/modules/admin/admin.service.ts

```diff
@@ -19,10 +19,14 @@ import {
 import { Bitflags } from '@momentum/bitflags';
 import { Role } from '@momentum/constants';
 import { DbService } from '../database/db.service';
+import { UsersService } from '../users/users.service';
 
 @Injectable()
 export class AdminService {
-  constructor(private readonly db: DbService) {}
+  constructor(
```

Comment:
No need for the `deleteUser` method here, just call `delete` in the `UsersService` directly from the AdminController. Otherwise we end up with a spaghetti of service methods calling each other and nothing else.